src/frontend/src/views/Data/components/TreeActionMenu/components/MoreActionsMenu/style.ts

```diff
@@ -1,13 +1,23 @@
 import styled from "@emotion/styled";
-import { getColors, getIconSizes, getSpaces } from "czifui";
+import { getColors, getIconSizes, getSpaces, Props } from "czifui";
+import EditIcon from "src/common/icons/IconEditSmall.svg";
 import TrashIcon from "src/common/icons/IconTrashCanSmall.svg";
 
-export const StyledText = styled.span`
-  ${(props) => {
+interface ExtraProps extends Props {
+  isRed?: boolean;
+}
+
+const doNotForwardProps = ["isRed"];
+
+export const StyledText = styled("span", {
+  shouldForwardProp: (prop) => !doNotForwardProps.includes(prop as string),
+})`
+  ${(props: ExtraProps) => {
+    const { isRed } = props;
     const colors = getColors(props);
 
     return `
-      color: ${colors?.error[600]};
+      color: ${isRed ? colors?.error[600] : "black"};
```

Comment:
This is a pain, but I _think_ we might actually not be using pure black for most text? I don't know well enough to be sure, but a quick check on my local shows our `<body>` having a blanket color of `rgba(0,0,0,.87)`, so a not-quite-black color.

There's two ways I can think of remedying this, but I don't really have a complete understanding of how we do theming with czifui. The easier option would be to just use the general text theme color instead of "black", but... I don't see `rgba(0,0,0,.87)` anywhere in our theme... _looking_... hooo, I think it might actually come from some other package called "semantic"? Yeah, looks like the CSS is pulling from that, and I see an import of `import "semantic-ui-css/semantic.min.css";` But that means we don't really directly set that color ourselves, so this option is kinda out...

The other option -- and I think pretty much the only reasonable one now -- is to have the CSS completely disappear when `isRed` is not present. That way it will inherit whatever the usual color would be without us needing to explicitly set it. Basically something like this:
```
  ${(props: ExtraProps) => {
    const { isRed } = props;
    if ( !isRed ) {
      return ''; // This might actually be better as null than empty string? I forget the nuances of the styled API
    }
    const colors = getColors(props);
    return `color: ${colors?.error[600]};`;
  }}
```

Reply:
It's challenging to see, but if you look at this screenshot, you can (just barely) make out the slight difference.
![image](https://user-images.githubusercontent.com/89553795/152582081-481eba48-5782-4915-848b-191e7fc08c66.png)

The "Selected" was explicitly set to `black`, while the text in the rows of the table (eg, "Orange County") is inheriting the general body value of `rgba(0,0,0,.87)`

Reply:
Considering we have other places where we're mixing in pure `black`, it's not a big deal here if you want to keep what you have, but it does speak to an underlying difficulty with how coloring and general app styles are being applied. Not sure if it's worth trying to figure out some months down the road or if we should just let it slide forever.

Reply:
ok resolved based on suggestion